apps/www/components/headers/ctaHeader/CtaHeader.tsx

```diff
@@ -36,12 +36,17 @@ export const CtaHeaderComponent: React.FC<
       <Container as="header" className={styles.appHeaderCta}>
         <nav className={styles.appTabs}>
           <ActiveLink href="/questions" activeClassName={styles.active}>
-            <a onClick={() => reportEvent('Lista pytań')} className={styles.appTabsTab}>
+            <a
+              data-cy="navigate-list"
```

Comment:
before you add too many selectors that use data-cy I reccomend to give a try to this lib:
https://testing-library.com/docs/cypress-testing-library/intro

you could simply select this part with:
`cy.findByText("Lista pytań")`

Reply:
Thanks for pointing this library. It looks very useful nevertheless I would stay with this `data-cy` for consistency.
Also this is small app so potential change won't be hard to implement